### DISCUSSION TITLE
Welcome to GitHub Discussions

### DISCUSSION THREAD

**Discussion by jeromegamez** [kreait/firebase-php#517] [General]
# 👋  Hey everyone! Welcome to GitHub Discussions!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Remember that this is a community we build together. Please be welcoming and open minded.

To get started, comment below with an introduction of yourself and tell us about what you do with the Firebase Admin SDK.

**Comment by guru1112001** [comment]
Hey  i want to ask that is the Kreait is been modify for the new firebase change " **change Migrate from legacy FCM APIs to HTTP v1** " please can someone answer ?

**Reply by jeromegamez** [reply]
The SDK uses the new HTTP V1 APIs since they were introduced, yes.
